test/endOfROSCAUnitTest.js

```diff
@@ -36,6 +36,23 @@ contract('end of ROSCA unit test', function(accounts) {
       }
     }
 
+	  it("throws if there are no more fees to be collected",
```

Comment:
nit: the test suite currently stars with the "normal" flow, then checks for the edge cases. We might want to keep that, by moving the new test past the "normal-flow" ones,